components/classes.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { FunctionComponent } from "react";
+import { Class } from "../models/classes";
+import ClassCard from "../components/ClassCard";
+import styles from "../styles/Components.module.css";
+
+type Props = {
+  classes: Class[] | undefined;
```

Comment:
I dont think we need this undefined here. If this page is going to render, it should always render with the class's loaded in